src/app/icd/ICDManager.cpp

```diff
@@ -168,10 +223,13 @@ void ICDManager::UpdateOperationState(OperationalState state)
         }
 #endif
 
+        // Going back to Idle, all check-in messages are sent
+        mICDSenderPool.ReleaseAll();
+
         CHIP_ERROR err = DeviceLayer::ConnectivityMgr().SetPollingInterval(slowPollInterval);
-        if (err != CHIP_NO_ERROR)
+        if (err != CHIP_NO_ERROR && err != CHIP_ERROR_NOT_IMPLEMENTED)
```

Comment:
Not sure we want to suppress the not implemented error...

Reply:
Changed on PR https://github.com/project-chip/connectedhomeip/pull/30552